Cargo.toml

```diff
@@ -4,15 +4,13 @@ version = "0.1.0"
 description = "Cargo wrapper for developing Rust-based Nintendo 3DS homebrew apps"
 repository = "https://github.com/Meziu/cargo-3ds"
 license = "MIT"
-authors = [ "Andrea Ciliberti <meziu210@icloud.com>" ]
+authors = ["Andrea Ciliberti <meziu210@icloud.com>"]
 edition = "2018"
```

Comment:
If it isn't too much of an issue, you could switch to edition 2021.

Reply:
Done. No changes were required to the code.